MovingBlocks/TeraNUI
# feat: Switch analytics config to use a zip dependency instead of local files

Applies a slightly tweaked version of the setup we use in the engine. Trying to improve the setup a bit each time, like where the files go.

To test: `gradlew check` should work

Drawbacks: the Checkstyle config isn't injected into IntelliJ as an option for the IDE plugin 😞 We used to hack that in via `idea` task but the import-via-Gradle approach bypasses that. Could probably check part of the `.idea` directory in, but if we start doing that for multiple projects then that just seems like trading one instance of duplication (store analytics config files in every repo) for another (maintain `.idea` files in every repo)

Wonder if a custom Gradle plugin could help somehow, we already do have one started ...

Only applying to `v1.x` for now, but if merged we should update elsewhere too.
